TCL4 Data Management/utm-tcl4-dmp-fet-uas.yaml

```diff
@@ -91,65 +83,171 @@ definitions:
+        format: float
+        example: 15.85        
+
+      verticalSpeed_ftPerMin:
         description: >-
-          Relative humidity near the UAS at this timestamp.  Percentage.
+          Vertical speed in feet per minute, negative indicates descent. (ft/min)
         type: number
         format: float
-        minimum: 0.0
-        maximum: 100.0
-        example: 62.88
-      airspeed:
+        example: 15.85        
+          
+      roll_deg:
         description: >-
-          airspeed in ft/sec
+          Roll (deg). Negative roll indicates left.
         type: number
         format: float
-        example: 22.91
-      vertical_speed:
+        example: 15.85        
+        
+      pitch_deg:
         description: >-
-          vertical speed in ft/sec
+          Pitch (deg). Negative pitch indicates nose down.
         type: number
         format: float
-        example: 1.27
-      roll_decimal_degrees:
+        example: 15.85        
+        
+      yaw_deg:
         description: >-
-          roll measured in decimal degrees.
+          Yaw (deg). Zero-degree yaw is North.
         type: number
         format: float
-        example: 29.46
-      pitch_decimal_degrees:
+        example: 15.85        
+        
+      rollRate_degPerSec:
         description: >-
-          pitch measured in decimal degrees.
+          Roll Rate (deg/s). Negative indicates leftward rate.
         type: number
         format: float
-        example: 2.81
-      yaw_decimal_degrees:
+        example: 15.85        
+        
+      pitchRate_degPerSec:
         description: >-
-          yaw measured in decimal degrees.
+          Pitch Rate (deg/s). Negative indicates downward rate.
         type: number
         format: float
-        example: 100.49
-      roll_rate:
+        example: 15.85        
+        
+      yawRate_degPerSec:
         description: >-
-          rate of change of roll angle in degrees/sec.
+          Yaw Rate (deg/s). Negative indicates counter-clockwise rate.
         type: number
         format: float
-        example: 0.37
-      pitch_rate:
+        example: 15.85        
+        
+      hdop_nonDim:
         description: >-
-          rate of change of roll angle in degrees/sec.
+          Number of GPS satellites to which the aircraft has line-of-sight to (whether acquired or not by the GPS receiver).
         type: number
         format: float
-        example: 0.37
-      yaw_rate:
+        example: 15.85        
+        
+      vdop_nonDim:
         description: >-
-          rate of change of roll angle in degrees/sec.
+          VDOP: Vertical dilution of precision of GPS constellation.
         type: number
         format: float
-        example: 0.37
+        example: 15.85        
+        
+      numGpsSatellitesInView_nonDim:
+        description: >-
+          Number of GPS satellites to which the aircraft has line-of-sight to (whether acquired or not by the GPS receiver).
```

Comment:
So is this calculated by a separate system from the UAS platform?  If so, they need to run this tool against the UAS position for the course of the operation either during or after flight, right?  This may be a difficult dependency that hampers getting operational data.  Please correct my thinking if incorrect.

Reply:
yes that is my understanding from tcl3.  In this way of thinking, we would remove this data element from the model and create another one - of similar structure, that might include some other ones, perhaps related only to the time-scale on which it can be provided to us.  Do you agree?

Reply:
Take a look at https://github.com/nasa/utm-docs/commit/43256612185511a7da80ead5bebc0c8fbdd86928
made an offline model and put it here.  Now in our expected data we can put a different expected timestamp on this model - thoughts?

Reply:
commented there.